SwiftWisdomTests/Rx/CollectionVariableTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+import RxSwift
+@testable import SwiftWisdom
+
+class CollectionVariableTests: XCTestCase {
```

Comment:
There's actually a whole suite of tests for this in the original library, I just never updated them to swift3. Since we're absorbing the library, we should absorb the tests too: https://github.com/pepibumur/CollectionVariable/blob/master/CollectionVariableTests/CollectionVariableTests.swift